discovery-service/src/test/java/org/zowe/apiml/discovery/eureka/RefreshablePeerEurekaNodesTest.java

```diff
@@ -88,7 +88,6 @@ void setUp() {
     void givenEurekaNodeUrl_thenCreateNode() throws NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException {
         when(serverConfig.getPeerNodeTotalConnections()).thenReturn(100);
         when(serverConfig.getPeerNodeTotalConnectionsPerHost()).thenReturn(10);
-        when(replicationClientAdditionalFilters.getFilters()).thenReturn(Collections.emptyList());
```

Comment:
Related to a previous comment, is this `replicationClientAdditionalFilters` not used anymore?